base/shell/explorer/traywnd.cpp

```diff
@@ -615,20 +658,26 @@ class CTrayWindow :
 
         case ID_SHELL_CMD_TILE_WND_H:
             appbar_notify_all(NULL, ABN_WINDOWARRANGE, NULL, TRUE);
+            if (m_Arrangement == NONE) BackupWindowPos();
```

Comment:
Should it remember the first or the last?
And I don't think the style of this file is to have a complete if-statement on one line..

Reply:
The last state before it's TILED or CASCADED, therefore, only if current arrangement is "NONE".
Coding style fixed (all occurences)